lib/puppet/provider/splunk_output/ini_setting.rb

```diff
@@ -7,9 +7,9 @@
   def self.file_path
     case Facter.value(:osfamily)
     when 'windows'
-      'C:\Program Files\Splunk\etc\system\local\outputs.conf'
+      'C:\Program Files\Splunk\etc\apps\search\local\outputs.conf'
```

Comment:
This one was to match inputs.conf which when I use splunk forwarder-add <splunk-server> it puts the forwarder in search\local\inputs.conf so I thought that  was the right file. I could just change it for the inputs.conf though and revert everythign else back to system if that would work?